astrolabe/cli.py

```diff
@@ -0,0 +1,484 @@
+# Copyright 2020-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from pprint import pprint
+
+import click
+
+import astrolabe.commands as cmd
+import astrolabe.docgen as docgen
+from atlasclient import AtlasClient
+from atlasclient.configuration import CONFIG_DEFAULTS as CL_DEFAULTS
+from astrolabe.docgen import (
+    tabulate_astrolabe_configuration, tabulate_client_configuration)
+from astrolabe.spec_runner import MultiTestRunner, SingleTestRunner
+from astrolabe.configuration import (
+    CLI_OPTION_NAMES as OPTNAMES,
+    CONFIG_DEFAULTS as DEFAULTS,
+    CONFIG_ENVVARS as ENVVARS,
+    TestCaseConfiguration)
+from astrolabe.utils import ClickLogHandler
+
+
+__logger__ = logging.getLogger(__name__)
```

Comment:
Why `__logger__` and not simply `logger`?

Reply:
changed to LOGGER